src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using NLog;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Definitions.UNIT3D;
+using NzbDrone.Core.Messaging.Events;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class LatTeam : Unit3dBase
+    {
+        public override string Name => "LatTeam";
+        public override string Language => "es";
```

Comment:
change to format as `{ISO 639-1}-{ISO 3166-1 alpha-2}`

Reply:
hey there, the thing is that idk in which country they are based in and there are like 20 ones that speak spanish in the americas, should i pick a random?